tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py

```diff
@@ -0,0 +1,170 @@
+import os
+import subprocess
+
+from ruamel import yaml
+
+import great_expectations as ge
+
+context = ge.get_context()
+
+# parse great_expectations.yml for comparison
+great_expectations_yaml_file_path = os.path.join(
+    context.root_directory, "great_expectations.yml"
+)
+with open(great_expectations_yaml_file_path, "r") as f:
+    great_expectations_yaml = yaml.safe_load(f)
+
+stores = great_expectations_yaml["stores"]
+pop_stores = ["checkpoint_store", "evaluation_parameter_store", "validations_store"]
+for store in pop_stores:
+    stores.pop(store)
+
+actual_existing_expectations_store = dict()
```

Comment:
@NathanFarmer we should use `{}` universally, unless concatenating several dictionaries into one.  Thanks.

Reply:
✅